frontend/src/components/Template/NavBar.css

```diff
@@ -4,7 +4,7 @@
 
 .signup-btn {
   position: absolute;
-  right: 10px;
+  right: 10%;
```

Comment:
Your buttons are not consistent. Your signup btn is 10% while login is 10px
Also avoid using absolute as much as possible if you can make it work without it.
You may want to use flex box instead.